utils/src/main/kotlin/ArchiveUtils.kt

```diff
@@ -184,8 +184,13 @@ fun File.unpack7Zip(targetDirectory: File) {
  * a [prefix] is specified, it is added to the file names in the ZIP file.
  * If not all files shall be added to the archive a [filter] can be provided.
  */
-fun File.packZip(targetFile: File, prefix: String = "", filter: (Path) -> Boolean = { true }) {
-    require(!targetFile.exists()) {
+fun File.packZip(
+    targetFile: File,
+    prefix: String = "",
+    overwrite: Boolean = false,
+    filter: (Path) -> Boolean = { true }
+) {
+    require(overwrite || !targetFile.exists()) {
```

Comment:
Nit: Don't you think swapping the conditions ready a bit more natural?

Reply:
I find the current order natural. It also does the cheaper check first.

Reply:
Ok, doing the cheaper check first makes sense.